htmresearch/regions/TemporalPoolerRegion.py

```diff
@@ -44,6 +44,10 @@ def _getPoolerClass(name):
         " union, unionMonitored" % (name))
 
 
+def _getParentSpatialPoolerClass():
+    return UnionTemporalPooler.__bases__[0]
```

Comment:
Would be better if you called `_getPoolerClass()` so this isn't hard coded for the union implementation, and return all the bases (not just the first in the list).